dsl/src/main/scala/asobu/dsl/util/RecordOps.scala

```diff
@@ -0,0 +1,31 @@
+package asobu.dsl.util
+
+import shapeless._
+import shapeless.labelled.FieldType
+
+object RecordOps {
+
+  type FieldKV[K, +V] = K with ValueTag[K, V]
+  trait ValueTag[K, +V]
+
+  trait FieldKVs[L <: HList] extends DepFn0 with Serializable { type Out <: HList }
+
+  object FieldKVs {
+
+    type Aux[L <: HList, Out0 <: HList] = FieldKVs[L] { type Out = Out0 }
+
+    implicit def hnilFieldKV[L <: HNil]: Aux[L, HNil] =
+      new FieldKVs[L] {
+        type Out = HNil
+        def apply(): Out = HNil
+      }
+
+    implicit def hlistFieldKVs[K, V, T <: HList](implicit
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [You should not name methods after their defining object](https://www.codacy.com/app/iHeartRadio/asobu/file/2261096926/issues/source?bid=3037259&fileBranchId=3178877#l23)